sdcm/sct_provision/user_data_objects/sshd.py

```diff
@@ -26,6 +26,6 @@ def is_applicable(self) -> bool:
     @property
     def script_to_run(self) -> str:
         script = configure_sshd_script()
-        script = configure_ssh_accept_rsa()
+        script += configure_ssh_accept_rsa()
```

Comment:
How did Azure tests work without sshd configuration?

Reply:
We are not using it that much, so we didn't notice any issues related to that, those are most configuration for handling better cases of reconnection and such